Templates/default/Package.stencil

```diff
@@ -1,8 +1,8 @@
 // swift-tools-version: 5.9
```

Comment:
I'd update this one to `// swift-tools-version: 6.0` instead of adding `@preconcurrency`